src/components/table/table.tsx

```
@@ -103,6 +99,16 @@ export class Table {
     @Prop({ mutable: true })
     public activeRow: RowData;
 
+    /**
+     * Array of rows that have been active
+     */
+    private activeRowHistory: Tabulator.RowComponent[] = [];
+
+    /**
+     * Maximum number of rows to keep in the active row history
+     */
+    private static readonly MAX_ACTIVE_ROW_HISTORY = 2;
```

ESLint

[error] No magic number: 2.

(no-magic-numbers)